src/explore-education-statistics-admin/src/pages/AdminDashboard.tsx

```diff
@@ -1,29 +1,68 @@
-import AdminDashboardApprovedForPublication from '@admin/pages/prototypes/components/AdminDashboardApprovedForPublication';
-import AdminDashboardNeedsWork from '@admin/pages/prototypes/components/AdminDashboardNeedsWork';
-import AdminDashboardReadyForApproval from '@admin/pages/prototypes/components/AdminDashboardReadyForApproval';
+import React, { useContext, useEffect, useState } from 'react';
 import RelatedInformation from '@common/components/RelatedInformation';
 import Tabs from '@common/components/Tabs';
 import TabsSection from '@common/components/TabsSection';
-import React from 'react';
-import { RouteChildrenProps } from 'react-router';
+import { Dictionary } from '@common/types';
 import { LoginContext } from '@admin/components/Login';
-import { Authentication } from '@admin/services/PrototypeLoginService';
-import AdminDashboardPublications from '@admin/components/AdminDashboardPublications';
 import DummyPublicationsData from '@admin/pages/DummyPublicationsData';
-import Accordion from '@common/components/Accordion';
+import { Publication } from '@admin/services/publicationService';
+import AdminDashboardPublicationsTab from '@admin/components/AdminDashboardPublicationsTab';
+import groupBy from 'lodash/groupBy';
 import Link from '../components/Link';
 import Page from '../components/Page';
 
-const BrowseReleasesPage = ({ location }: RouteChildrenProps) => {
+const AdminDashboardPage = () => {
+  const emptyPublicationsArray: Publication[] = [];
+
+  const [myPublications, setMyPublications] = useState(emptyPublicationsArray);
+
+  const [inProgressPublications, setInProgressPublications] = useState(
+    emptyPublicationsArray,
+  );
+
+  const authentication = useContext(LoginContext);
+
+  const loadInitialData = () => {
```

Comment:
Could just inline this as a lambda in `useEffect`

Reply:
It started life off as a lambda, but I wanted to just be a bit explicit about what was going on with useEffect() in this instance for other people's sakes, hence putting a name to it.  If loading initial data is generally all we use useEffect() for, I'm happy to put it back as a lambda - what do you think?

Reply:
Personally it looks pretty obvious what's going on :wink: If you're worried about separation of concerns, you can just call `useEffect` multiple times to separate out different blocks of logic.

Reply:
Fairy nuff.  Done!